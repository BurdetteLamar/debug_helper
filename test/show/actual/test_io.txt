---
IO (message='test_io'):
  autoclose?: true
  binmode?: false
  closed?: false
  tty?: false
