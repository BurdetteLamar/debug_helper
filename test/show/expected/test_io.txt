---
IO (message='test_io'):
  IO#autoclose?: true
  IO#binmode?: false
  IO#closed?: false
  IO#tty?: false
